src/Hashids.php

```diff
@@ -79,7 +100,10 @@ class Hashids implements HashidsInterface
      *
      * @return void
      */
-    public function __construct($salt = '', $minHashLength = 0, $alphabet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890')
+    public function __construct(
+        $salt = Hashids::DEFAULT_SALT,
```

Comment:
Because Hashids is not a final class, I think you could use `static`.
Like that if anyone would like to extends `Hashid`, he could add his own constant without changing code for example :sunglasses: